src/cls/rbd/cls_rbd.cc

```diff
@@ -3694,7 +3694,7 @@ int metadata_remove(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
 
```

Comment:
fix the typo `metdata_remove`  -> `metadata_remove` in previous line [3693](https://github.com/ceph/ceph/pull/25436/files#diff-e393a69c5ca0d5ea740c0d60d68e55f5R3693) if you wish ?